k8s/resources/study/config/voltage-init-server-config.yml

```diff
@@ -3,4 +3,7 @@ network:
 
 # Nb of concurrent processes - should be in-sync with voltageinit.run.voltageinitGroup queue concurrency
 computation-local:
-  available-core: 2
\ No newline at end of file
+  available-core: 2
+
```

Comment:
// TODO: to remove when upgrade to powsybl-optimizer v0.9.0 is done

Reply:
yes